options/components/webBottomContent/WebBottomContent.jsx

```diff
@@ -36,8 +37,21 @@ function WebUrlNewList() {
               className="mr-2 inline-block w-3 h-3"
               src={faviconURL(url.url)}
             />
-            {url.title}
-            {hasSearchResult ? <HighlightKeyword url={url} /> : ""}
+            {hasSearchResult &&
+            keyword &&
+            userSelectSearchValue === "키워드 검색" ? (
+              <HighlightKeyword url={url} />
+            ) : hasSearchResult &&
+              keyword &&
+              userSelectSearchValue === "제목 검색" ? (
+              <HighlightTitleKeyword url={url} />
+            ) : hasSearchResult &&
+              keyword &&
+              userSelectSearchValue === "제목 + 키워드 검색" ? (
+              <HighlightKeyword url={url} />
+            ) : (
+              url.title
+            )}
```

Comment:
```suggestion
            {isHighlightKeyword(SELECT_VALUE_STATE[0]) ? (
              <HighlightKeyword url={url} />
            ) : isHighlightKeyword(SELECT_VALUE_STATE[1]) ? (
              <HighlightTitleKeyword url={url} />
            ) : isHighlightKeyword(SELECT_VALUE_STATE[2]) ? (
              <HighlightKeyword url={url} />
            ) : (
              url.title
            )}
```
SELECT_VALUE_STATE를 context에 저장하신 걸로 확인이 됩니다. 따라서 위의 형태와 같이 활용가능할 것 같습니다. 또한, 추후에 유지보수를 고려하여 반복되는 변수를 사용하는 코드를 함수화 하는 것도 좋을 것 같아 보입니다. 개인적인 의견이므로, 전체적인 상황에 맞게 자율적으로 적용해주시면 좋을 것 같습니다.

Reply:
유지보수를 생각한다면 추후 관리하기 좋은 로직같습니다. 제안에 감사드리며 말씀 참고하여 적용 할 수 있도록 하겠습니다.